src/rules/typescript/original.ts

```diff
@@ -151,6 +151,16 @@ export = {
     // https://typescript-eslint.io/rules/no-empty-interface
     '@typescript-eslint/no-empty-interface': 'error',
 
+    // Disallow duplicate constituents of union or intersection types.
+    // https://typescript-eslint.io/rules/no-duplicate-type-constituents
+    '@typescript-eslint/no-duplicate-type-constituents': ['error', {
```

Comment:
This rule requires type information to run, so it should go to `typescript-requiring-type-checking/original.ts`.